src/app/app.module.ts

```diff
@@ -19,7 +19,6 @@ import {
   MatIconModule,
   MatSelectModule,
   MatExpansionModule,
-  MatTooltip,
```

Comment:
Why is this removed?

Reply:
Should probably be done in its own commit (I had missed this). But the only that should be needed to be included from material ui in app.module.ts are *Module* files. See e.g. MatExpansionModule, MatSelectModule etc. I assume we have a MatTooltipModule somewhere that allows us to use MatTooltip in our components.